portal/static/js/src/modules/TnthAjax.js

```diff
@@ -272,6 +272,47 @@ export default { /*global $ */
             }
         });
     },
+    "getOrg": function(orgId, params, callback) {
+        callback = callback || function() {};
+        if (!orgId) {
+            callback({error: i18next.t("Organization id is required.")});
+            return false;
+        }
+        if (sessionStorage[`orgData_${orgId}`]) {
+            callback(JSON.parse(sessionStorage[`orgData_${orgId}`]));
+            return true;
+        }
+        params = params || {};
+        this.sendRequest("/api/organization/"+orgId, "GET", "", params, function(data) {
+            if (!data.error) {
+                $(".get-orgs-error").html("");
+                sessionStorage.setItem(`orgData_${orgId}`, JSON.stringify(data));
+                callback(data);
+            } else {
+                var errorMessage = i18next.t("Server error occurred retrieving organization/clinic information.");
```

Comment:
If these appear as messages to endusers, I think these could be generic "server error" error messages. If you want to include details about the specific API that's returning errors, maybe just include the API base URL (rather than a human-readable description that must be translated)?

Reply:
thanks, ivan, fixed.